lectures/unpleasant.md

```diff
@@ -501,11 +494,13 @@ mystnb:
 plot_path([80, 100], msm)
 ```
 
-Figure  {numref}`fig:unpl1` summarizes outcomes of  two experiments that convey   messages of 
+{numref}`fig:unpl1` summarizes outcomes of  two experiments that convey   messages of 
 Sargent and Wallace's **unpleasant monetarist arithmetic** {cite}`sargent1981`.
 
 * An open market operation that reduces the supply of money at time $t=0$ reduces  the price level at time $t=0$
 
 * The lower is the post-open-market-operation money supply at time $0$, lower is the price level at time $0$.
 
-* An open  market operation that reduces the post-open-market-operation money supply at time $0$ also **lowers** the rate of return on money $R_u$ at times $t \geq T$ because it brings  a higher gross-of-interest government deficit that must be financed by printing money (i.e., levying an inflation tax) at time $t \geq T$.
+* An open  market operation that reduces the post-open-market-operation money supply at time $0$ also *lowers* the rate of return on money $R_u$ at times $t \geq T$ because it brings  a higher gross-of-interest government deficit that must be financed by printing money (i.e., levying an inflation tax) at time $t \geq T$.
```

Comment:
I would suggest we change `post-open-market-operation` to `post open market operation`

similar comment for `gross-of-interest`